contracts/nois-proxy/src/msg.rs

```diff
@@ -42,13 +42,11 @@ pub enum ExecuteMsg {
     },
     // Withdraw the given amount to the withdrawal address
     Withdaw {
-        amount: Coin,
-        address: String,
-    },
-    // Withdraw all available balance of this token to the withdrawal address
-    WithdawAll {
         denom: String,
-        address: String,
+        // If None Withdraw all available balance of the denom to the withdrawal address
+        amount: Option<Uint128>,
+        // If None then funds will be sent to the community pool
+        address: Option<String>,
```

Comment:
This feels scary to me. If you forget setting the Options in JSON, you don't get an error but they will be parsed as None. So forgetting `address` or a typo in the field name means the funds go to the wrong address.